nvflare/app_common/executors/client_api_launcher_executor.py

```diff
@@ -147,7 +147,7 @@ def prepare_config_for_launch(self, shareable: Shareable, fl_ctx: FLContext):
         config_dict[ConfigKey.PIPE_CLASS] = self._pipe.__class__.__name__
         config_dict[ConfigKey.PIPE_ARGS] = self._get_external_pipe_args()
         config_dict[ConfigKey.TOTAL_ROUNDS] = total_rounds
-        config_dict[ConfigKey.SITE_NAME] = fl_ctx.get_identity_name()
+        config_dict[ConfigKey.IDENTITY] = fl_ctx.get_identity_name()
```

Comment:
is it "get_identity()" (without _name)? (according to the import code below `from nvflare.client import get_identity as get_identity`)

Reply:
@ZiyueXu77 thanks for the review.

The "get_identity_name" method is from "fl_ctx".

So two different methods.

Do you think it is better to have both methods calling the same name ?